src/compas/artists/artist.py

```diff
@@ -88,13 +132,12 @@ class Artist(object):
 
     __ARTISTS_REGISTERED = False
 
-    AVAILABLE_CONTEXTS = ["Rhino", "Grasshopper", "Blender", "Plotter"]
-    CONTEXT = None
+    KNOWN_CONTEXTS = ["Rhino", "Grasshopper", "Blender", "Plotter", "Viewer"]
```

Comment:
perhaps this can be populated during the plugin discovery? then automatically all registered artist plugins add their context to the "known contexts"...

Reply:
Removed this class attribute as `Artist.ITEM_ARTIST` is already populated with discovered contexts by the plugins.